test/iht/testhelpers/MessagesTidierTest.scala

```diff
@@ -618,14 +661,36 @@ class MessagesTidierTest extends UnitSpec with FakeIhtApp {
       }
     }
 
+    "compareMessageFileKeys" must {
+
+      "not fail when mocked english and welsh message files have the same keys" in {
+        val diff: Set[String] = mockedMessagesTidier.compareMessageFileKeys()
+        assert(diff == Set.empty)
+      }
+
+      "fail when mocked english and welsh message files do not have the same keys" in {
+        val diff: Set[String] = mockedMessagesTidier.compareMessageFileKeysWelshFailure
+        val diff2 = mockedMessagesTidier.compareMessageFileKeysEnglishFailure
+        assert(diff != Set.empty && diff2 != Set.empty)
+      }
+
+      "not fail with real messages files" in {
+        pending
+        val result = MessagesTidier.compareMessageFileKeys()
+        assert(result.isEmpty, "\n \n There are message keys missing from messages.en and/or messages.cy - " +
+          "see the file /home/" + System.getProperty("user.name") + "/Desktop/missingKeysAndValues.txt for more info")
```

Comment:
I've changed the output file to a .csv and removed Desktop. There are other uses of Desktop in the MessagesTidierTest file, so we may want to look at those separately if it is an issue for non-Ubuntu machines.